frontend/packages/operator-lifecycle-manager/src/components/descriptors/reference/reference.md

```diff
@@ -30,14 +32,33 @@
   5. [k8sPhase](#5-k8sPhase)
   6. [k8sPhaseReason](#6-k8sPhaseReason)
   7. [k8sResourcePrefix](#7-k8sResourcePrefix)
-
+---------------------------------------
 
 #### [DEPRECATED Descriptors](#deprecated-descriptors-1)
+  15. [fieldGroup](#fieldGroup)
+  16. [arrayFieldGroup](#arrayFieldGroup)
+
 
 ## specDescriptors
-A specDescriptor is for describing the property of a `spec` field in your Custom Resource. Each specDescriptor provides UI widget(s) for creating/mutating CR instance on different views (i.e. CREATION, DISPLAY, and MODIFY views) in the OpenShift Console.
+A specDescriptor is for describing the property of a `spec` field in your Custom Resource. Each entry should contain the following:
```

Comment:
```suggestion
A specDescriptor is for describing the `spec` stanza in your Custom Resource. Each entry should contain the following:
```

Reply:
Since the `spec` **stanza** means the full `spec` block in the manifest, let's keep that sentence as-is. Thanks Jon!